src/renderer/families/ethereum/GasPriceField.js

```diff
@@ -26,7 +26,7 @@ const FeesField = ({ onChange, account, transaction, status }: Props) => {
 
   const onGasPriceChange = useCallback(
     gasPrice => {
-      onChange(bridge.updateTransaction(transaction, { gasPrice }));
+      onChange(bridge.updateTransaction(transaction, { gasPrice, feesStrategy: "advanced" }));
```

Comment:
same. i know it's old pattern but updateTransaction should be used (if we pass it as props)